chainer/training/util.py

```diff
@@ -31,6 +28,8 @@ def get_trigger(trigger):
         object made from ``trigger``.
 
     """
+    from chainer.training.triggers import interval_trigger
```

Comment:
Importing this at module-level would cause cyclic import:
```
>>> import chainer
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "chainer/__init__.py", line 23, in <module>
    from chainer import training  # NOQA
  File "chainer/training/__init__.py", line 1, in <module>
    from chainer.training import extensions  # NOQA
  File "chainer/training/extensions/__init__.py", line 11, in <module>
    from chainer.training.extensions.micro_average import MicroAverage  # NOQA
  File "chainer/training/extensions/micro_average.py", line 3, in <module>
    from chainer.training import util
  File "chainer/training/util.py", line 1, in <module>
    from chainer.training.triggers import interval_trigger
  File "chainer/training/triggers/__init__.py", line 2, in <module>
    from chainer.training.triggers.early_stopping_trigger import EarlyStoppingTrigger  # NOQA
  File "chainer/training/triggers/early_stopping_trigger.py", line 5, in <module>
    from chainer.training import util
ImportError: cannot import name util
```
I wrote a comment about this.